queries/python.scm

```diff
@@ -209,7 +291,16 @@
 ) @class @className.domain
 
 (module) @className.iteration @class.iteration
-(module) @statement.iteration @value.iteration @name.iteration
+(module) @statement.iteration
+
+;; This is a hack to handle the case where the entire document is a `with` statement
+(
+  (module
+    (_) @_statement
+  ) @value.iteration @name.iteration
+  (#not-type? @_statement "with_statement")
+)
```

Comment:
make sure we still have iteration scope when there is a top-level with statement and other statments as well. eg:

```py
with aaa:
    pass

bbb = ccc
ddd = eee
```

Reply:
confirmed